### PR TITLE
Add Builder for Scope

### PR DESCRIPTION
**Pull request by kageiit**

### CODE REVIEW

File: buildSrc/src/main/java/com/uber/okbuck/core/model/android/AndroidTarget.java [hunk 1]
```diff
@@ -157,22 +157,24 @@ public AndroidTarget(Project project, String name) {
...
-        getUnitTestVariant() != null ? getSources(getUnitTestVariant()) : ImmutableSet.of(),
-        getJavaResources(getUnitTestVariant()),
-        getJavaCompilerOptions(getUnitTestVariant()));
+    Scope.Builder builder = Scope.builder(getProject());
+    UnitTestVariant unitTestVariant = getUnitTestVariant();
+    if (unitTestVariant != null) {
+      builder.configuration(unitTestVariant.getRuntimeConfiguration());
+      builder.sourceDirs(getSources(getUnitTestVariant()));
+      builder.javaCompilerOptions(getJavaCompilerOptions(unitTestVariant));
+    }
```
Comment (raviagarwal7) [inline buildSrc/src/main/java/com/uber/okbuck/core/model/android/AndroidTarget.java]: missing `resources`

Comment (kageiit) [reply buildSrc/src/main/java/com/uber/okbuck/core/model/android/AndroidTarget.java]: good catch. mind fixing in a follow up?

Comment (kageiit) [reply buildSrc/src/main/java/com/uber/okbuck/core/model/android/AndroidTarget.java]: Fixed in https://github.com/uber/okbuck/commit/1d25f8918783a3870d46709159f6101fc94fbf8a

Comment (kageiit) [reply buildSrc/src/main/java/com/uber/okbuck/core/model/android/AndroidTarget.java]: for null, we return empty map which we dont need to handle anymore as its the default. Will clean up that code later

**Pull request merged**